README.rst

```diff
@@ -43,11 +43,11 @@ Why would I use this instead of Django's `manage.py test` command?
 Running your test suite with pytest-django allows you to tap into the features
 that are already present in pytest. Here are some advantages:
 
-* `Manage test dependencies with pytest fixtures. <http://pytest.org/latest/fixture.html>`_
+* `Manage test dependencies with pytest fixtures. <http://pytest.org/en/latest/fixture.html>`_
 * Less boilerplate tests: no need to import unittest, create a subclass with methods. Write tests as regular functions.
```

Comment:
Anyway, the final link would be https://docs.pytest.org/en/latest/fixture.html - so it should be used here when going with this workaround probably.